src/components/Utility/CardHeader/CardHeader.css

```diff
@@ -0,0 +1,6 @@
+.cardheader-container {
+    height: calc(0.875rem + 20px);
```

Comment:
What's the logic for setting the height of this? And what's the `calc` function doing? Why not just leave the height unset, and let the content fill it, then control the space around the font with padding?

Reply:
Just controlling the height manually since it'll be the same everywhere. Not doing it this way leads to a container that takes up too much space (tbh - not 100% on why), so this was an easier way of doing it 👍🏻 You can try pulling this and removing the height and you'll see that it acts up.

The "calc" function calculates the correct height even if you mix units. So I can have it set the height to the font size + 10px on each side for padding. Would not work great if something were to take up 2 lines of text space, but nothing in this project will.